src/main/assembly/dist/cfg/application.properties

```diff
@@ -5,3 +5,4 @@ auth.ldap.url=ldap://localhost
 auth.ldap.user=cn=ldapadmin,dc=dans,dc=knaw,dc=nl
 auth.ldap.password=changeme
 audio-video.ffprobe=/usr/bin/ffprobe
+dans.doi=10.17026/
```

Comment:
Remember to add extra application properties also to dtap

Reply:
also done